SubmitHGCalPGun.py

```diff
@@ -271,22 +273,23 @@ def submitHGCalProduction(*args, **kwargs):
         processCmd('mkdir -p '+outDir+'/std/')
         processCmd('mkdir -p '+outDir+'/jobs/')
 
-
   # prepare dir for GSD outputs locally or at EOS
     if (opt.LOCAL):
         processCmd('mkdir -p '+outDir+'/'+opt.DTIER+'/')
         recoInputPrefix = 'file:'+currentDir+'/'+opt.inDir+'/'+previousDataTier+'/'
         if (opt.DQM): processCmd('mkdir -p '+outDir+'/DQM/')
     elif opt.eosArea:
-        processCmd(eosExec + ' mkdir -p '+opt.eosArea+'/'+outDir+'/'+opt.DTIER+'/');
+        if opt.DTIER != 'ALL':
+            processCmd(eosExec + ' mkdir -p '+opt.eosArea+'/'+outDir+'/'+opt.DTIER+'/');
+        else:
+            processCmd(eosExec + ' mkdir -p '+opt.eosArea+'/'+outDir+'/NTUP/');
```

Comment:
Will it work, if you only create the NTUP directory for the "ALL" case? Shouldn't you create all (GSD, RECO, DQM, NTUP) directories in this case?

Reply:
it should work because the the GSD and RECO steps as it are not copied in the output dir, only the NTUP is. I have added a comment about this in the README.